config/config.toml.example

```diff
@@ -66,6 +66,11 @@ delay-clean-table-lock = 0
 # Maximum number of the splitting region, which is used by the split region statement.
 split-region-max-num = 1000
 
+# alter-primary-key is used to control alter primary key feature. Default is false, indicate the alter primary key feature is disabled.
+# If it is true, we can add the primary key by "alter table", but we may not be able to drop the primary key.
+# In order to support "drop primary key" operation , this flag must be false and the table does not have the pkIsHandle Flag.
```

Comment:
@zimulala According to your answer, `drop primary key` is only available when alter-primary-key is true and the table's pkIsHandle is false. Plz update the description.